lib/GroupBackend.php

```diff
@@ -111,7 +111,7 @@ public function getUserGroups($uid): array {
 	 * Returns a list with all groups
 	 */
 	public function getGroups($search = '', $limit = -1, $offset = 0): array {
-		return $offset === 0 ? [$this->groupName] : [];
+		return $offset == 0 ? [$this->groupName] : [];
```

Comment:
What the hell?
This is typed as an int, it should make no difference.
Remove the change or fix the phpdoc please.

Reply:
it should be an int, and is typed as such in the interface phpdoc, but gets passed as `null` anyway.

I'll fix the place that is passing the `null`s, but that only helps once new server versions are released.

Reply:
https://github.com/nextcloud/server/pull/36592